nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -185,7 +185,6 @@ let
       ''}
 
       ${optionalString cfg.recommendedGzipSettings ''
-        # https://docs.nginx.com/nginx/admin-guide/web-server/compression/
```

Comment:
Please keep this comment, it's for developers of this file and it is also useful for people reading the configuration file.

Reply:
If activate Brotli compression, then this comment looks redundant:
```
  ssl_ciphers ...;
  brotli on;
  brotli_static on;
  brotli_comp_level 5;
  brotli_window 512k;
  brotli_min_length 256;
  brotli_types application/atom+xml ... ;
  # https://docs.nginx.com/nginx/admin-guide/web-server/compression/
  gzip on;
  gzip_static on;
  gzip_vary on;
  gzip_comp_level 5;
  gzip_min_length 256;
  gzip_proxied expired no-cache no-store private auth;
  gzip_types ... l;
```

Reply:
In that case, let's move it outside of the configuration file, but please, keep it just above line 187.

Reply:
Then we will have 2 duplicates - on line 186 and in parameter description.

Reply:
It's fine to have duplication because people who read the parameter description can read it from search.nixos.org and those who will read the source code might stumble more easily on line 186 rather than the parameter description. It's more of a "let's make it really easy for people to read what's going on and have pointers to how to understand this configuration" thing IMHO

Reply:
And, if I move it a little higher, with compression disabled, the configuration will look like this:
```
  ssl_protocols TLSv1.2 TLSv1.3;
  ssl_ciphers ...;
  # https://docs.nginx.com/nginx/admin-guide/web-server/compression/
  # $connection_upgrade is used for websocket proxying
  map $http_upgrade $connection_upgrade {

```
```

Reply:
I meant to move it outside of the configuration file, only in the Nix code